drivers/gpio/gpio_b9x.c

```diff
@@ -679,7 +790,7 @@ static void gpio_b9x_irq_connect_4(void)
 	static struct gpio_b9x_data gpio_b9x_data_##n;			    \
 									    \
 	DEVICE_DT_INST_DEFINE(n, gpio_b9x_init,				    \
-			      NULL,					    \
+			      PM_DEVICE_DT_INST_GET(n),					    \
```

Comment:
#if (defined CONFIG_PM_DEVICE && (defined(CONFIG_BOARD_TLSR9518ADK80D_RETENTION) \
|| defined(CONFIG_BOARD_TLSR9528A_RETENTION)))
otherwise NULL

Reply:
not necessary, working properly without this change.
Take a look at zephyr/drivers/serial/uart_b9x.c. The same way done there